src/patternfly/upgrade-examples/LoginUpgradeExamples/docs/code.md

```diff
@@ -0,0 +1,26 @@
+## Overview
+
+When converting PatternFly 3 components to PatternFly 4 components, you must also take into consideration the layouts and sizings that PatternFly 3 utilized from Bootstrap 3.
+
+## Usage
+
+| PF3 Class | Applied To | PF4 Class | Applied To | Outcome |
+| -- | -- | -- | -- | -- |
+| `.login-pf` | `<body>` | `.pf-c-login` | `<div>` | Parent container that holds various login components. |
+| `.container-fluid`| `<div>` | `.pf-c-login__container`| `<div>` | Primary container that holds the login component itself. |
+| `.login-pf-page-header` | `<header>` |`.pf-c-login__header` | `<header>` | Header container that sits outside of the login component. |
+| `.login-pf-brand`| `<img>` | `.pf-c-brand` | `<img>` | Patternfly logo brand image tag. |
+| `.login-pf-header` | `<header>` | `.pf-c-login__main-header` | `<header>` | Header container that sits inside of the login component. |
+| `.card-pf` > `<form>` | `<form>` | `.pf-c-login__main` | `<main>` | Main container that holds the form section of the login component. |
+| `.card-pf` > `<form>` | `<form>` | `.pf-c-form` | `<form>` | Primary login form. |
+| `.form-group` | `<div>` | `.pf-c-form__group`| `<div>` | Parent container that holds form label and input field containers. |
+| `.sr-only` | `<label>` | `.pf-c-form__label` | `<label>` | Inner form label. |
```

Comment:
You can use `label` instead of `.sr-only`. Though I'm curious if we want to be this granular in the upgrade example, or if we're just documenting the main structural elements of the login components.

Reply:
I honestly don't know how granular we want to be. I placed this line of code in there because PF4 seems more specific. And when i say specific, I mean the class is `**pf-c**-something` specific.

Reply:
I'm just wondering if the list should be limited to things that are either `.login-pf*` or `.pf-c-login*` - document all of that. But if there are buttons and things in the forms, should we document those? Maybe @dgutride can clarify?

Reply:
I'd agree with @mcoker  - we don't really need to be that granular (and we should avoid documenting PF3 too much here.  We just really want to show clear paths forward for main classes.

Reply:
Reduced multiple lines of code that are not essential components to the login example.